lib/web_ui/lib/src/engine/window.dart

```diff
@@ -99,8 +99,10 @@ class EngineWindow extends ui.Window {
   /// Simulates clicking the browser's back button.
   Future<void> webOnlyBack() => _browserHistory.back();
 
+  String _defaultRouteName;
```

Comment:
It took me a while to realize (found the `??=` in the code) that this is lazy-initialized and cached route name. I think a comment would be useful.

Reply:
Done.